index.js

```diff
@@ -133,8 +133,10 @@ class LedgerBridgeKeyring extends EventEmitter {
       },
       ({ success, payload }) => {
         if (success) {
```

Comment:
Is it possible to add a unit test for each of the true and false cases for `updateHdk`? (Or update existing tests of methods that call `unlock` to ensure that they would fail without this change)

Reply:
yep I added test cases